mm2src/mm2_main/src/rpc/lp_commands/one_inch/rpcs.rs

```diff
@@ -193,7 +196,10 @@ mod tests {
             "derivation_path": "m/44'/1'",
             "chain_id": 1,
```

Comment:
redundant chain_id?

Reply:
IMHO for tokens configs we should get chain_id from the platform coin config (to have the single source of it)

Reply:
Yep, this one is actually not needed.

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/pull/2425/commits/b7806a7be834857a353bcac51f22498e02de9050